CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # Changelog
 
+## 1.0.1
+
+- Publish package to NPM during CI `publishWorkflow.yml`
+- Add Jean Duthon as maintainer
+- Change pull request template to include checklist items
+
 ## 1.0.0
 
 - First public release
```

Comment:
1. Yes that's a good point, I'll make it `1.0.0`
2. I've seen plenty of changelogs in open source where they mention adding/removing maintainers and general maintenance work. If a file in the repository changes, it should be reflected in the changelog. Remember we have a maintainers file that reflects this. For this PR, I can remove these items since everything will fall under initial release for `1.0.0`, but that's how I'm thinking about it for the future. Thoughts?

Reply:
Interesting!
Yeah works for me, even more so if it's standard practice in some other OSS work!

I was basing my thoughts on what I had seen myself.

I also guess that this should be a rather "boring" (in the good sense) repo once it gets off the ground, so changelogs are likely not to be too crazy, so being over descriptive should be ok.
We can always course correct if it's not 👍